server/controller/Meetup.js

```diff
@@ -0,0 +1,160 @@
+import Meetups from '../model/Meetup';
+
+class Meetup {
+
+   /**
+    * Create A Meetup
+    * @param {object} req 
+    * @param {object} res
+    * @returns {object} meetup object 
+    */
+   static createMeetup(req, res) {
+      let tags = req.body.tags instanceof Array ? req.body.tags : [req.body.tags]
```

Comment:
Consider declaring this as a constant since it doesn't seem re-assigned

Reply:
done